lib/features/modeling/Modeling.js

```diff
@@ -425,6 +431,17 @@ Modeling.prototype.replaceShape = function(oldShape, newShape, hints) {
   return context.newShape;
 };
 
+Modeling.prototype.replaceRoot = function(oldShape, newShape) {
```

Comment:
Having both apis, replace _and_ update root could be confusing. My proposal is to to add some documentation for the API to highlight the differences.

Reply:
I agree and I also thought about this.

* We already have the `canvas#updateRoot` API, hence I would like to follow this naming for the Modeling-API and command-name. Hence `modeling#updateRoot` is kind of set into stone from my perspective (otherwise we would get inconsistent)
* `modeling#replaceRoot` follows the same pattern as `modeling#replaceShape`, hence I would also vote for consistency here.

Hence I concluded to go for consistency despite of the fact that these two API methods might be confused with each other.

I added documentation to make this clearer.

(Alternative could be `modeling#setRoot` instead of `modeling#updateRoot`.)